packages/legacy-workbench/src/css/ttyg/agent-settings-modal.css

```diff
@@ -38,12 +38,53 @@
     color: var(--primary-color-dark);
 }
 
-.agent-settings-modal .agent-settings-form .provider-name {
-    padding-left: 5px;
+.agent-settings-modal .agent-settings-form .context-size .input-with-unit {
+    position: relative;
+}
+
+.agent-settings-modal .agent-settings-form .context-size .input-with-unit input {
+    padding-right: 3.5em;
```

Comment:
This is OK now, because `tokens` in french is `jetons` and the space is enough for the word to fit in. The better approach would be to lay-out the content in a flex-ible way so that the engine to be able to adjust block dimensions appropriately.
You can leave it now, but pay attention to such things in future